plugins/@grouparoo/google-sheets/src/lib/sheet-import/recordProperties.ts

```diff
@@ -0,0 +1,46 @@
+import { RecordPropertiesPluginMethod } from "@grouparoo/core";
+import { parseProfileProperties } from "@grouparoo/csv/dist/lib/shared/parseProfileProperties";
```

Comment:
Is this the write way to share or should this plugin really export some things?

Reply:
that's the way the other plugins are using (eg. clickhouse using MySQL).